templates/jobs/rollback-release.yml

```diff
@@ -7,7 +7,7 @@
 #   - KUBE_NAMESPACE: in which namespace the release was installed
 .helm-rollback-template:
   # We use the .post stage https://docs.gitlab.com/ee/ci/yaml/#stages
```

Comment:
Remove also the comment related to the `.post` stage.

Reply:
Changed the comment in:
# We use the deploy stage https://docs.gitlab.com/ee/ci/yaml/#stages